### DISCUSSION TITLE
Placement - groupings - How can I rename "Content" card?

### DISCUSSION THREAD

**Discussion by PiemP** [OrchardCMS/OrchardCore#13534] [Q&A] [answered]
I have see this discussion #11894 , but I don't have understand how can I rename the "Content" card?

**Comment by PiemP** [comment]
OK, I have found my issue. I'm using a content type with a `ListPart` and the `ListPartNavigationAdmin` go in grouping with the other Shapes and, without a key, go automatically in a card named "Content". With this placement rule I can correctly rename the "Content" card: 

```
{
  "ListPart_ContainerId":
  [
    {
      "place": "Content:0%Navigation;0",
      "displayType": "Edit",
      "contentType": [
        "Article"
      ]
    }
  ],
  "ListPartNavigationAdmin":
  [
    {
      "place": "Content:0%Navigation;0",
      "differentiator": "ListPart-ListPartNavigationAdmin",
      "contentType": [
        "Article"
      ]
    }
  ]
}
```

**Comment by ns8482e** [comment, marked as answer]
Correct, Placement will help you rename the tab. However remember! , you need to place all shapes to new name, if a shape/part is not placed using placement then default place will be `Content` and Content tab will show up again.
